spec/models/spree_avatax/shared_spec.rb

```diff
@@ -11,23 +11,24 @@
 
     context 'when the order is taxable' do
       let(:order) { create(:shipped_order, line_items_count: 1) }
-      it { should be true }
+      it { is_expected.to be true }
     end
 
     context 'when the order has no line items' do
       let(:order) { create(:order) }
-      it { should be false }
+      it { is_expected.to be false }
     end
 
     context 'when the order has no ship address' do
-      let(:order) { create(:shipped_order, line_items_count: 1, ship_address: nil) }
-      it { should be false }
+      let(:order) { create(:shipped_order, line_items_count: 1) }
+      before { order.ship_address = nil }
```

Comment:
It is not impossible for an order not to have a shipping address. However, the order created by this test is invalid. (Because it is a shipped order, and as such is invalid without a shipping address.)

I'll look through the order factories and see if I can find one that's better suited for the test.